README.md

```diff
@@ -140,7 +142,7 @@ Example:
 
 - [ ] **responsive design**
 - [ ] **demo project**
-- [ ] **personal profilo**
+- [ ] **capstone project**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Your design is very nice you did a great job :+1: but there are still some problems like you not adding appropriate credit to the author please kindly add it in the `Acknowledgements` section of your `README` file. check the following example :point_down: 

![Screenshot from 2023-03-16 00-15-10](https://user-images.githubusercontent.com/45226848/225419904-e6935286-ecfa-44bf-bf58-9099ccf94f9e.png)

Reply:
Fixed.